fedimint-core/Cargo.toml

```diff
@@ -53,6 +53,7 @@ macro_rules_attribute = "0.2.0"
 bitvec = "1.0.1"
 parity-scale-codec = { version = "3.6.9", features = ["derive"] }
 imbl = "2.0.3"
+backon = "0.4.3"
```

Comment:
Every new dep worries me, but seems small and very popular ... https://crates.io/crates/backon...

Reply:
maybe we just should fork it, just maths so should be pretty stable

Reply:
Nah. We pin it, it's good. It's not a big crate.

Reply:
pinned and added comment so someone doesn't approve dependabot :smile: